doc/src/compute_modify.rst

```diff
@@ -20,7 +20,7 @@ Syntax
          N = # of extra degrees of freedom to subtract
        *extra* syntax is identical to *extra/dof*, will be disabled at some point
        *dynamic/dof* value = *yes* or *no*
-         yes/no = do or do not re-compute the number of degrees of freedom (DOF) contributing to the temperature
+         yes/no = do or do not re-compute the number of degrees of freedom (dof) contributing to the temperature
```

Comment:
Since "DOF" is an initialism, I prefer it to be in uppercase or at least "DoF".

Reply:
The documentation was split on this; some used "dof" while others used "DOF"; I went with the first one I found, which was "dof," but I could change them all to DOF if that is preferable.

Reply:
It is certainly *my* preference. I cannot speak for the other LAMMPS developers.